src/wallet/rpcdump.cpp

```diff
@@ -814,6 +814,20 @@ UniValue ProcessImport(CWallet * const pwallet, const UniValue& data, const int6
         const bool watchOnly = data.exists("watchonly") ? data["watchonly"].get_bool() : false;
         const std::string& label = data.exists("label") && !internal ? data["label"].get_str() : "";
 
+        if (data.exists("privkey")) {
+            const UniValue& privkey = data["privkey"];
+            if (data.exists("scriptPubKey"))           throw JSONRPCError(RPC_INVALID_PARAMETER, "Must provide either a privkey or a scriptPubKey");
+            if (strRedeemScript != "")                 throw JSONRPCError(RPC_INVALID_PARAMETER, "Redeem script not supported for privkey import");
+            if (pubKeys.size() > 0 || keys.size() > 0) throw JSONRPCError(RPC_INVALID_PARAMETER, "Keys/pubkeys not supported for privkey import");
+            if (watchOnly)                             throw JSONRPCError(RPC_INVALID_PARAMETER, "Privkeys cannot be imported watch-only");
+            if (!ImportPrivateKey(pwallet, privkey, label, timestamp)) {
+                throw JSONRPCError(RPC_WALLET_ERROR, "The wallet already contains the given private key");
+            }
+            UniValue result = UniValue(UniValue::VOBJ);
```

Comment:
In commit "[wallet] [rpc] Add privkey support to importmulti",

Nit, `UniValue result(UniValue::VOBJ);`